__tests__/db.test.js

```diff
@@ -35,29 +36,87 @@ describe("App", () => {
           .get("/api/topics")
           .expect(200)
           .then(({ body }) => {
-            console.log(body.topics, " <<<< my bodyyy");
             body.topics.forEach((element) => {
               let objKeys = Object.keys(element);
-              expect(objKeys.includes("slug")).toBe(true);
-              expect(objKeys.includes("description")).toBe(true);
+              if (objKeys.length > 0) {
+                expect(objKeys.includes("slug")).toBe(true);
+                expect(objKeys.includes("description")).toBe(true);
+              }
             });
           });
       });
     });
   });
 
   describe("GET /api", () => {
-    test("Responds with a status of 200", () => {
-      return request(app).get("/api").expect(200);
-    });
     test("Response returns an object", () => {
+      //Edit this test description
+      return fs
+        .readFile(`endpoints.json`, "utf-8")
```

Comment:
you can just statically require the endpoints.json in to your test file instead of using readFile